mlflow/tracking/_model_registry/client.py

```diff
@@ -205,6 +205,18 @@ def create_model_version(
             self.store._await_model_version_creation(mv, await_creation_for)
         return mv
 
+    def copy_model_version(self, src_mv, dest_name):
+        """
+        Copy a model version from one registered model to another.
```

Comment:
```suggestion
        Copy a model version from one registered model to another as a new model version.
```

Can we add `as a new model version` for clarity?